bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/ReservedProperties.java

```diff
@@ -87,6 +87,8 @@ private ReservedProperties() {
     public static final String PN_MINIMUM = "minimum";
     public static final String PN_EXCLUSIVE_MINIMUM = "exclusiveMinimum";
     public static final String PN_EXCLUSIVE_MAXIMUM = "exclusiveMaximum";
+    public static final String PN_EXCLUDE_MINIMUM = "excludeMinimum";
+    public static final String PN_EXCLUDE_MAXIMUM = "excludeMaximum";
     public static final String PN_EXCLUDE_MINIMUM_CHECK = "excludeMinimumCheck";
```

Comment:
What is difference between PN_EXCLUDE_MINIMUM and PN_EXCLUDE_MINIMUM_CHECK ?

Reply:
These are old values, which was not reviewed correctly, now we need to maintain it to support old content structures